src/main/java/net/sf/jsqlparser/statement/alter/AlterExpression.java

```diff
@@ -44,7 +45,7 @@
   private String ukName;
   private Index index = null;
   private String constraintName;
-  private boolean constraintIfExists;
+  private boolean usingIfExists;
```

Comment:
Did you consider adding this field to the Alter class instead of merging it with constraintIfExist? that would also make the deparsing much more simple

Reply:
Thank you for your feedback, you are right of course: The whole `ALTER TABLE ...` statement needs to get overhauled -- but I did not have a time for this yesterday and just rushed a fix to make it work for you.

At the same time you are very knowledgeable and would make a great contributor. I stand ready to provide you with any assistance for a first PR.